src/System.Runtime.CompilerServices.Unsafe/System.Runtime.CompilerServices.Unsafe.sln

```diff
@@ -0,0 +1,35 @@
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
This file should be deleted. We don't add slns per library in this repo.

Reply:
Yep. I'm aware. I just haven't cleaned it up properly after migration.